testing/unit-tests/how-to-execute-tests.md

```diff
@@ -35,3 +35,40 @@ vendor/bin/phpunit --debug -c tests/Unit/phpunit.xml tests/Unit/PrestaShopBundle
 {{% notice tip %}}
 Hint: `--debug` without `--filter` gives a nice list to filter with.
 {{% /notice %}}
+
+## Checking for code coverage in phpunit tests
+
+To check if you covered everything in your tests cases its best to run the tests with [phpunit coverage report](https://phpunit.readthedocs.io/en/9.5/code-coverage-analysis.html).
+
+First get your [environment up and running]({{< relref "/8/basics/installation/localhost" >}}). 
+
+Then, [install and enable Xdebug](https://xdebug.org/docs/install).
+
+Finally, run your tests with coverage enabled:
+
+```bash
+XDEBUG_MODE=coverage vendor/bin/phpunit --coverage-text -c tests/Unit/phpunit.xml tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php
+```
+
+{{% notice tip %}}
+You may use a [Dockerized environment to run the project]({{< relref "/8/basics/installation/install-with-docker" >}}).
```

Comment:
Apparently the test is failing because this reference is not found

Reply:
thanks, i left a comment (in the PR desc) about the necessity to merge #1661 before this one since i reference a pending PR 👍